README.md

```diff
@@ -17,6 +17,15 @@ A web application for providing access to Digital Objects in SDR Storage.
   ./bin/test.sh
   ```
 
+### Note for osx users
+the current bundle requires eventmachine 1.0.7. This won't build on a modern mac
+without specifying where the ssl libraries are:
+
+```
+gem install eventmachine -v '1.0.7' -- --with-cppflags=-I/usr/local/opt/openssl/include
+```
+
```

Comment:
This is a way out of date version of the library.  The only reason you would encounter it is because it's in the Gemfile.lock of this project.  That is why it's not in the documentation of Eventmachine.  This is a problem for developers on the sdr-services-app, so I insist we leave it here.